src/target_info.rs

```diff
@@ -65,15 +65,21 @@ fn check_processor_target_compatability(cores: &[Core], elf_path: &Path) {
     let matches = match core_type {
         CoreType::Armv6m => target == "thumbv6m-none-eabi",
         CoreType::Armv7m => target == "thumbv7m-none-eabi",
+        CoreType::Armv7a => target == "thumbv7a-none-eabi",
         CoreType::Armv7em => target == "thumbv7em-none-eabi" || target == "thumbv7em-none-eabihf",
         CoreType::Armv8m => {
             target == "thumbv8m.base-none-eabi"
                 || target == "thumbv8m.main-none-eabi"
                 || target == "thumbv8m.main-none-eabihf"
         }
-        CoreType::Riscv => return, // NOTE(return) Since we do not get any info about instruction
-                                   // set support from probe-rs we do not know which compilation
-                                   // targets fit.
+        // NOTE(return) Since we do not get any info about instruction
+        // set support from probe-rs we do not know which compilation
+        // targets fit.
+        CoreType::Riscv => return,
+        _ => {
```

Comment:
```suggestion
        CoreType::Armv8a => {
```

I'd prefer to not use the wildcard (`_`), but rather handle all variants explicitly. If we handle them explicitly we will get a compile error when they add a new version, so we won't overlook that.

The only variant missing is `Armv8a`. Since there is no rustc target for that (as far as I know), I'd opt for handling it as you are already doing (printing a warning and returning).